lifecycle-application/pom.xml

```
@@ -40,7 +40,7 @@
             </activation>
             <properties>
                 <!-- please keep Mandrel and RHBQ version compatible -->
-                <quarkus.platform.version>2.11.2.Final</quarkus.platform.version>
+                <quarkus.platform.version>2.13.0.Final</quarkus.platform.version>
```

```suggestion
                <quarkus.platform.version>2.13.4.Final</quarkus.platform.version>
```

How to not forget about this :/ ?

---

> How to not forget about this :/ ?

For now I've added item to https://issues.redhat.com/browse/QUARKUS-2454 
We need also some natural reminder before test window starts